core/src/camera.rs

```diff
@@ -84,7 +84,7 @@ fn camera_controller(
         }
 
         // Get the player's body rect
-        let rect = body.collider_rect(transform.translation);
+        let rect = body.aabb(*transform);
```

Comment:
for those that dont know what aabb is, is there a way for us to document this for new developers to better understand? Maybe just a small comment